beacon-chain/core/helpers/committee.go

```diff
@@ -348,6 +349,12 @@ func UpdateCommitteeCache(state *stateTrie.BeaconState, epoch uint64) error {
 
 // UpdateProposerIndicesInCache updates proposer indices entry of the committee cache.
 func UpdateProposerIndicesInCache(state *stateTrie.BeaconState, epoch uint64) error {
+	// The cache uses the block root at the last epoch slot as key. (e.g. for epoch 1, the key is root at slot 31)
+	// Which is the reason why we skip genesis epoch.
+	if epoch == params.BeaconConfig().GenesisEpoch {
```

Comment:
```suggestion
	if epoch <= params.BeaconConfig().GenesisEpoch {
```
If GenesisEpoch is not guarenteed to be 0, you need to use less than / equal.